addons/rivet/devtools/dock/settings.tscn

```diff
@@ -136,62 +80,6 @@ layout_mode = 2
 [node name="DeployNamespaceSelector" parent="VBoxContainer/Deployment Fieds" instance=ExtResource("4_k8i13")]
 unique_name_in_owner = true
 layout_mode = 2
-namespaces = [{
-"create_ts": "2023-11-19T05:46:56.378Z",
-"display_name": "Production",
-"name_id": "prod",
-"namespace_id": "215bd313-b4c7-4932-b9bd-64f8f09b7fe8",
-"version": {
-"create_ts": "2023-11-19T05:46:56.171Z",
-"display_name": "0.0.1",
-"version_id": "73881376-21fa-4fb2-93f2-2f76fb3bac19"
-},
-"version_id": "73881376-21fa-4fb2-93f2-2f76fb3bac19"
-}, {
```

Comment:
Just out of curiosity, where did these changes come from?

Reply:
These changes are intended. That data comes from engine, when the script was running in the editor's scene preview, before `prepare` function on screens. Currently, we have `prepare` function, and screen's data is only populated when switching in plugin window in editor.